mne/time_frequency/tests/test_spectrum.py

```diff
@@ -386,3 +387,31 @@ def test_spectrum_kwarg_triaging(raw):
         raw.plot_psd(axes=axes)
     # `ax` is the correct legacy param name
     raw.plot_psd(ax=axes)
+
+
+def test_spectrumarray_raw(raw):
+    """Test SpectrumArray for Raw-derived spectra."""
+    spect = raw.compute_psd()
+    spect2 = SpectrumArray(
+        data=spect.get_data(), freqs=spect.freqs, info=spect.info, method=spect.method
+    )
+    assert_array_equal(spect.get_data(), spect2.get_data())
+    assert_array_equal(spect.freqs, spect2.freqs)
```

Comment:
should be possible to just do
```
    assert spect == spect2
```
because we define `__eq__` on these... though perhaps you're right that they won't/can't/needn't pass that stricter test?

Reply:
I tried and it failed, I think because you don't pass the provenance in the constructor, that's going to make them not identical